.gitlab-ci.yml

```diff
@@ -56,12 +56,13 @@ create_key:
 # STATIC ANALYSIS
 
 analysis:ktlint:
-  tags: [ "runner:main" ]
-  image: $CI_IMAGE_DOCKER
+  tags: [ "runner:docker" ]
   stage: analysis
   timeout: 30m
+  image: "registry.ddbuild.io/mobile/images/ktlint:v0.1.0"
   script:
-    - GRADLE_OPTS="-Xmx2560m" ./gradlew :ktlintCheckAll --stacktrace --no-daemon
+    - echo "Using Ktlint version $(ktlint --version)"
```

Comment:
it also means we have to keep local `ktlint` version in sync with the version used in the image (because the rules may be different)?

Reply:
Yes, exactly.
In a second step, we might find a way to ensure this condition or make it easier to upgrade/be in sync. Any ideas?

Reply:
I guess the only way is to just build a new docker image with a different tag (or have a fat docker image with multiple ktlint versions embedded).